src/components/swapv2/SingleTokenInfo.tsx

```diff
@@ -225,15 +230,15 @@ const SingleTokenInfo = ({
           className="desc"
           ref={ref}
           dangerouslySetInnerHTML={{
-            __html: isSeeMore
-              ? description.replace(/<[^>]+>/g, '') // plain text
-              : description.replaceAll('\r\n\r\n', '<br><br>'),
+            __html: description.replaceAll('\r\n\r\n', '<br><br>'),
           }}
         ></div>
-        {seeMoreStatus !== SeeStatus.NOT_SHOW && (
-          <SeeMore onClick={toggleSeeMore}>See {isSeeMore ? 'more' : 'less'}</SeeMore>
-        )}
       </DescText>
+      {seeMoreStatus !== SeeStatus.NOT_SHOW && (
+        <SeeMore onClick={toggleSeeMore} isSeeMore={isSeeMore}>
+          See {isSeeMore ? 'more' : 'less'}
```

Comment:
Please change this to:
```
{isSeeMore ? <Trans>See more</Trans> : <Trans>See less</Trans>}
```

This will help for translation

Reply:
updated.